int-test/tests/login.py

```diff
@@ -0,0 +1,62 @@
+from appium import webdriver
```

Comment:
If the libraries are just as good, doing JavaScript for the mobile tests is a great strategy, since the fewer languages one needs to learn, the better.

Already RN has JavaScript/iOS/Android, that's enough things to know :)